onnx/optimizer/passes/eliminate_nop_monotone_argmax.h

```diff
@@ -0,0 +1,56 @@
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
+#include "onnx/optimizer/passes/optimize_pass.h"
+
+namespace ONNX_NAMESPACE {
+namespace optimization {
+
+const std::unordered_set<NodeKind> monotone_node_no_axis_kind{kLog,
+                                                              klog,
```

Comment:
Here we should use kLog, kExp, kSqrt. And remove klog, kexp, ksqrt from intern_strings